lib/cucumber_factory/factory.rb

```diff
@@ -210,8 +214,12 @@ def unquote(string)
         string[1, string.length - 2]
       end
 
+      def file_value_to_path(string)
+        string[6, string.length - 7]
```

Comment:
Ouch, this is not readable at all.

```suggestion
        # file paths are marked with a special keyword and enclosed with quotes.
        # Example: file:"/path/image.png"
        # This will extract the path (/path/image.png) from the text fragment above.
        string[6, string.length - 7]
```

Reply:
```suggestion
        unquote string.sub(/\Afile:/, '')
```